packages/odk/src/Utils.js

```diff
@@ -0,0 +1,95 @@
+/**
```

Comment:
We should remove this comment from the actual adaptor (it's correct to leave it in the template!)